palindrom

```diff
@@ -0,0 +1,34 @@
+function pal(start , end , thisword){
+
+   
+    if( thisword[start] != thisword[end] ){
+        console.log(thisword[start]);
+        console.log(thisword[end]);
+        return false;
+    }
+   
+    if(start != end){
+        pal(start+1 , end-1 , thisword);
+        console.log("plaindrom");
+    }
+    
+    return true;
+}
+
+function isPalindromeFunction(str)
```

Comment:
We usually don't don't include the word "function" in the names of our functions, it's redundant. Just "isPalindrome" would suffice.